src/components/common/Style.tsx

```diff
@@ -7,22 +11,56 @@ const Style = ({
   content?: string
   children?: React.ReactNode[]
 }) => {
-  let css = content
-  if (!css && typeof children?.[0] === "string") {
-    css = children[0].trim()
+  const cssStateTs = useRef(new Date(0))
+  const [css, setCss] = useState("")
+
+  // Init CSS
+  const initCss = () => {
+    if (content) {
+      setCss(content)
+    } else if (typeof children?.[0] === "string") {
+      setCss(children[0].trim())
+    } else {
+      setCss("") // Nothing
+    }
   }
-  if (!css) {
-    return null
+
+  // Preview CSS
+  const previewCSSChannel = new BroadcastChannel("previewCSS")
```

Comment:
Component maybe re-render and this will create a new `BroadcastChannel`, is this the expected behavior?

Reply:
Nope, thanks for the remind :D